README.md

```diff
@@ -5,9 +5,9 @@ Small repo done in cab ride
 A simple repo used to track progress of codechef monthly contests from JAN15-JUNE17 (Can be increased or decresed by modifying code)
 
 ## What to do?
-* Install virtual env
+* Install virtual env 
 * Create an environment using virtual env `virtualenv envName -p python3`
-* Activate the environment and install the requirements `source bin/activate; pip3 install -r req.txt`
+* Activate the environment and install the requirements `source envName/bin/activate; pip3 install -r req.txt`
```

Comment:
Ah..... you keep project inside virtualenv not the other way around. So, it would be like
contest/ContestDone
Right now it is like contest/ContestDone/ContestDone/helper.py
But contest/ContestDone is like the root of the project where manage.py is, so I recomment it to be put at 
contest/ContestDone/helper.py Anyways, just keep the helper.py at the same place at manage.py I guess that much is clear? 
Since we will always be in that location and that is where we start the server and also make the probelms.txt file